### PR TITLE
Update packages for 3.5.0 GA

### PR DESCRIPTION
**Pull request by akofink**

### CODE REVIEW

**Comment by akofink** [comment]
~Blocked by https://github.com/Katello/katello/pull/7091 and a release of Katello to rubygems.~

**Comment by akofink** [comment]
[test]

**Comment by ehelms** [comment]
The `katello-installer` source is not git annexed.

File: repos/katello-repos.spec [hunk 1]
```diff
@@ -1,5 +1,3 @@
-%global prever .rc2
-
 Name:           katello-repos
 Version:        3.5.0
 Release:        2%{?prever}%{?dist}
```
Comment (ehelms) [inline repos/katello-repos.spec]: I reocmmend bumping the release here.

File: katello/katello.spec [hunk 2]
```diff
@@ -4,8 +4,6 @@
 %global homedir %{_datarootdir}/%{name}
 %global confdir common
 
-%global prever .rc2
-
 Name:       katello
 Version:    3.5.0
 Release:    1%{?prever}%{?dist}
```
Comment (ehelms) [inline katello/katello.spec]: I reocmmend bumping the release here.

File: katello-installer/katello-installer-base.spec [hunk 3]
```diff
@@ -2,8 +2,6 @@
 %undefine scl_prefix
 %global scl_ruby /usr/bin/ruby
 
-%global prever .rc2
-
 Name:    katello-installer-base
 Version: 3.5.0
 Release: 1%{?prever}%{?dist}
```
Comment (ehelms) [inline katello-installer/katello-installer-base.spec]: I reocmmend bumping the release here.

**Comment by akofink** [comment]
Updated release versions and changed binary files to annex index.

**Comment by akofink** [comment]
[test]

**Comment by ehelms** [comment]
[test]

**Comment by ehelms** [comment]
Looks to be passing now! There is one small but large change missing:

https://github.com/Katello/katello-packaging/commit/d51f40ae8d62ee82c6b31f5c8335058155ec43d8#diff-c9df48368001a35d665ab52a9382f333R8

This is adding in the 3.5 GPG key, updating the repos RPM for it, and then modifying the actual repositories to enable gpgcheck. The 3.5 key is:

```
-----BEGIN PGP PUBLIC KEY BLOCK-----
Version: GnuPG v1

mQENBFmgfYoBCAC9EEEMUfybb3eOF+SxSEcEIY3ZehL/YweJKBOgTHuC0NNAbg8q
5WR8ycA7XwQPxi3pizWUjUeAYoUxk5bKaTrh5tbBNLsb/0kxgzXwMQ4OEmEWjdPt
DLAyq5dmwxhvsQF+xOiN0RHrnE/ad7BKqJ5tI4rdrrQI3G1oAF4qWwFn6acrsiHz
YEuhNFlfCld03NNEqMc0LvgsO5+1Oqm9v0hqh2CX3ri4dlb5FwPKKRlvRiYHGQ7O
yhr6wKXHAMfP50n9NRevgAr7uzy1R87TtdGL9jCPxnvMABxkSiDAGXonvp0ZoI7u
mrhXBd2RfgZ8cRQ/UtffGKxY1r4ZJyrhOVRdABEBAAG0KkthdGVsbG8gMy41IDxm
b3JlbWFuLWRldkBnb29nbGVncm91cHMuY29tPokBPgQTAQIAKAUCWaB9igIbAwUJ
AsfqAAYLCQgHAwIGFQgCCQoLBBYCAwECHgECF4AACgkQ6RPNaSiE7O/6Ygf/f+vw
ezzcZteE3DLv9Hg7DXfH1tnGoETWRG2nq7WAfSLJxvxsCbYrNgdfTryIdKyIFuZd
AyKhKvup4Tch8Rs7NKFs44xKihDQ6a9CmpamGRAKzBxyoSTgGo+Ek1WWjGMV7gpt
IqaChvsV2DxBuzRwy8yftmFhDwHf2V/HjXwN7SsY3VSpj6ol0KX516CmWyt7vAXE
La5ebZBqM+/+r41qy83Jn39CqNYgqejAxJFZcXnCfY7KEzL/NgjmpUPH0kRkUWyc
T2LAdzH30ZE+1Jee85sBNnVI+bg6e0KaoTymmuMFNEQj3uAFuoN2bY1qUUd6hSJJ
ITDrqPkkMrg9+CoR2A==
=vvmA
-----END PGP PUBLIC KEY BLOCK-----
```

**Comment by akofink** [comment]
Updated

**Comment by ehelms** [comment]
ACK - merging and building

**Pull request merged**